files/justfiles/secureblue.just

```diff
@@ -500,6 +500,19 @@ audit-secureblue:
         print_status "$CHRONYD_TEST_STRING" "$STATUS_FAILURE"
     fi
 
+    SECUREDNS_TEST_STRING="Ensuring system DNS resolution is secure"
+    if systemctl is-active --quiet systemd-resolved; then
+        DNSSEC_STATUS="$(cat /etc/systemd/resolved.conf.d/10-securedns.conf | grep "DNSSEC")"
+        DOT_STATUS="$(cat /etc/systemd/resolved.conf.d/10-securedns.conf | grep "DNSOverTLS")"
+        if [[ "$DNSSEC_STATUS" == "DNSSEC=true" && "$DOT_STATUS" == "DNSOverTLS=true" ]]; then
+            print_status "$SECUREDNS_TEST_STRING" "$STATUS_SUCCESS"
+        else
+            print_status "$SECUREDNS_TEST_STRING" "$STATUS_FAILURE"
+        fi
+    else
+        print_status "$SECUREDNS_TEST_STRING" "$STATUS_WARNING"
```

Comment:
if systemd-resolved isn't active, this should be a fail, no?

Reply:
IMO we should just do a fail in that case, since we can't validate custom configurations